bin/run.sh

```diff
@@ -38,6 +38,6 @@ case $1 in
     latest-inventory-to-kinto ${@:2}
     ;;
   *)
-    usage
+    exec "$@"
```

Comment:
This is nifty. Any command that you run after `docker run  -it mozilla/buildhub` would be basically executed. E.g. `docker run  -it mozilla/buildhub bash` or `docker run  -it mozilla/buildhub ls /tmp`